### PR TITLE
update NEWS for 2.7.0

### PR DESCRIPTION
**Pull request by XinfengZhang**
prepare to branch out 2.7.0

Signed-off-by: Carl Zhang <carl.zhang@intel.com>

### CODE REVIEW

File: NEWS [hunk 1]
```diff
@@ -1,6 +1,19 @@
-libva NEWS -- summary of user visible changes.  2019-12-25
+libva NEWS -- summary of user visible changes.  2010-03-26
```
Comment (MilhouseVH) [inline NEWS]:
```suggestion
libva NEWS -- summary of user visible changes.  2020-03-26
```
?

And 2019->2020 below.

Comment (MilhouseVH) [reply NEWS]: Also to be consistent, the format of the date below should be 26.Mar.2020 not `March`

File: NEWS [hunk 2]
```diff
@@ -1,6 +1,19 @@
...
 Copyright (C) 2009-2019 Intel Corporation
 
+version 2.7.0 - 26.March.2019
```
Comment (dvrogozh) [inline NEWS]: kind of summary of comments below. I would suggest the following:

```
version 2.7.0 - 26.March.2019
* Add av1 decode interfaces: va_dec_av1.h
* Add 2 FourCC for 10bit RGB(without Alpha) format: X2R10G10B10 and X2B10G10R10.
* trace: av1 decode buffers trace
* trace: Add HEVC REXT and SCC trace for decoding.
* Return error when no va devices available [bug #???].
* Fix android build issue and remove some trailing whitespace [bug #???]
* Adjust call sequence to ensure authenticate operation is executed [bug #???]
```

File: NEWS (hunk 1 above)
Comment (XinfengZhang) [reply NEWS]: accept

File: NEWS [hunk 3]
```diff
@@ -1,6 +1,19 @@
...
+* trace: av1 decode buffers trace
+* trace: Add HEVC REXT and SCC trace for decoding.
+* add matrix_coefficients into VADecPictureParameterBufferAV1
+* change the variable name style of va_dec_av1
+* change the format of va_dec_av1.h to unix style
+* include va_dec_av1.h in makefile.am
```
[the remark this answers is not in the document]
Comment (XinfengZhang) [reply NEWS]: accept

File: NEWS [hunk 4]
```diff
@@ -1,6 +1,19 @@
...
+* Add av1 decode interfaces
+* Return error when no va devices available.
```
[the remark this answers is not in the document]
Comment (XinfengZhang) [reply NEWS]: yes, but there are no bug number, I changed this item to fix crashes on system without supported hardware by PR #369

File: NEWS [hunk 5]
```diff
@@ -1,6 +1,19 @@
-libva NEWS -- summary of user visible changes.  2019-12-25
+libva NEWS -- summary of user visible changes.  2010-03-26
 Copyright (C) 2009-2019 Intel Corporation
 
+version 2.7.0 - 26.March.2019
+* trace: av1 decode buffers trace
+* trace: Add HEVC REXT and SCC trace for decoding.
+* add matrix_coefficients into VADecPictureParameterBufferAV1
```
[the remark this answers is not in the document]
Comment (XinfengZhang) [reply NEWS]: it is apart of av1 decode interface, will combined them to one

**Comment by uartie** [comment]
You should update the copyright year, too, in NEWS.

File: NEWS [hunk 6]
```diff
@@ -1,6 +1,15 @@
-libva NEWS -- summary of user visible changes.  2019-12-25
+libva NEWS -- summary of user visible changes.  2020-03-26
 Copyright (C) 2009-2019 Intel Corporation
```
Comment (uartie) [inline NEWS]: You should update the copyright year, too.

File: NEWS [hunk 7]
```diff
@@ -1,6 +1,15 @@
-libva NEWS -- summary of user visible changes.  2019-12-25
+libva NEWS -- summary of user visible changes.  2020-03-26
```
Comment (dvrogozh) [inline NEWS]: Update copyright year in a next line, please.

**Pull request merged**